src/controllers/album.js

```diff
@@ -0,0 +1,94 @@
+const getDb = require('../services/db');
+
+exports.create = async (req, res) => {
```

Comment:
It would be good to establish how many spaces you are going to be using throughout the project, this file has a mix of 2 and 4 at different points. Rather than say the same thing repeatedly across this review, best bet would be to run a formatter over the whole project as its varied throughout